src/stylesheets/components/_buttons.scss

```diff
@@ -9,111 +9,40 @@ button {
   @include btn;
 }
 
-/*--- Default Button Toggle ---*/
-
-.btn--toggle {
-  border: 0;
-  display: none;
-  position: relative;
-  background-color: transparent;
-  padding-right: $base-spacing-unit-small;
-
-  &::after {
-    content: '';
-    display: inline-block;
-    position: absolute;
-    top: 50%;
-    right: 0;
-    vertical-align: middle;
-    border: $base-spacing-unit solid transparent;
-    border-bottom-width: 0;
-    border-top-color: $grey-1;
-    margin: -($base-spacing-unit / 2) 0 0 $base-spacing-unit-tiny;
-  }
-}
-
-.has-js .btn--toggle {
-  display: inline;
-}
-
-/*--- Button Toggle [within the header] ---*/
-
-header .btn--toggle {
-  top: $base-spacing-unit;
-  right: 0;
-  width: 57px;
-  height: 50px;
-  position: absolute;
-  background-color: $deep-purple;
-  padding: $base-spacing-unit $base-spacing-unit-small;
-  @media (max-width: $medium - 1) {
-    display: inline;
-  }
-
-  &:after {
-    display: none;
-  }
-  &:hover, &:focus, &:active {
-    outline: none;
-
-    svg[data-icon="search"] g {
-      fill: $grey-3;
-    }
-    svg[data-icon="close"] g {
-      stroke: $grey-3;
-    }
-  }
-}
-
-.has-js header .btn--toggle {
-  @include media($medium) {
-    display: none;
-  }
-}
-
-/*--- Button Toggle within components ---*/
+/*--- Dropdown buttons ---*/
 
 .dropdown,
 .dropdown--block {
-  .btn--toggle {
+  > button {
     margin: 0;
   }
-}
-
-.dropdown.open,
-.dropdown--block.open {
-  .btn--toggle::after {
+  > button::after {
+    content: '';
+  }
+  > button:not([class*="btn"]) {
```

Comment:
This can all go under the default `button` styles. `typography--light` can then be included to the button mixin